test/Coordinator.ts

```diff
@@ -182,7 +183,7 @@ describe("Coordinator Test suit", function () {
                 before(async function () {
                     /// Get initial state
                     borrowedLvUSDInPositionBeforeLeverage = await r.cdp.getLvUSDBorrowed(nftIdFirstPosition);
-                    console.log("borrowedLvUSDInPositionBeforeLeverage", borrowedLvUSDInPositionBeforeLeverage);
```

Comment:
When using `console.log(..)`, would we see it even if calling simple `npm run test`?

Reply:
yeah, no way to prevent that unless we override default console.log function which is probably not a good idea because it will impact other libs as well potentially